x-pack/plugin/frozen-indices/src/internalClusterTest/java/org/elasticsearch/index/engine/frozen/FrozenIndexTests.java

```diff
@@ -275,7 +274,9 @@ public void testDoubleFreeze() {
             client().execute(
                 FreezeIndexAction.INSTANCE,
                 new FreezeRequest("test-idx").indicesOptions(
-                    new IndicesOptions(EnumSet.noneOf(IndicesOptions.Option.class), EnumSet.of(IndicesOptions.WildcardStates.OPEN))
+                    IndicesOptions.newBuilder()
+                        .wildcardOptions(IndicesOptions.WildcardOptions.newBuilder().allowEmptyExpressions(false))
+                        .build()
```

Comment:
Yes, indeed the `ALLOW_NO_INDICES` was an option in the `EnumSet` of `IndicesOptions.Option`, since the flag is not there then this makes the `allowEmptyExpressions` false.